PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,11 @@
+## Thanks for starting a pull request on Material Components!
+
+#### Don't forget:
+- [ ] Are you sure you mean this repo? This repository is a reference to the [main Flutter repository](https://github.com/flutter/flutter). Code-level issues should be created there.
+- [ ] Link to GitHub issues it solves. ```closes #1234```
+- [ ] Sign the CLA bot. You can do this once the pull request is submitted.
```

Comment:
Suggestion: 'opened' instead of 'submitted'.  'Submitted' could mean either 'Open' or 'Merged', so it's clearer to use the language for just one.

Reply:
So true. Gonna fix on other platforms too.